lib/Dialect/OM/Transforms/FreezePaths.cpp

```diff
@@ -154,7 +154,12 @@ LogicalResult PathVisitor::processPath(PathOp path) {
   assert(topModule && "must have something?");
   auto *node = instanceGraph.lookup(topModule);
   while (!node->noUses()) {
-    auto module = node->getModule();
+    auto module = node->getModule<hw::HWModuleLike>();
+
+    // If the module is public, stop here.
+    if (module.getVisibility() == SymbolTable::Visibility::Public)
```

Comment:
FWIW should be able to just use, `isPublic()` .  Whatever you like better :+1: .

Reply:
oh nice, i like isPublic better